Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -53,12 +56,17 @@ extension MongoCollection {
         }
 
         /// Adds the `deleteOne` operation to a bulk write
+        ///
```

Comment:
nit: I think @jmikola and I decided a while back that for > 3 lines docstrings we should use `/** ... */` instead

Reply:
Fixed.